proto/interchain_security/ccv/consumer/v1/genesis.proto

```diff
@@ -13,28 +13,31 @@ import "tendermint/abci/types.proto";
 // GenesisState defines the CCV consumer chain genesis state
 message GenesisState {
   Params params = 1 [ (gogoproto.nullable) = false ];
-  string provider_client_id = 2;  // empty for a completely new chain
-  string provider_channel_id = 3; // empty for a completely new chain
+  string provider_client_id = 2;  // empty for a new chain, filled in on restart.
+  string provider_channel_id = 3; // empty for a new chain, filled in on restart.
   bool new_chain = 4; // true for new chain GenesisState, false for chain restart.
   // ProviderClientState filled in on new chain, nil on restart.
   ibc.lightclients.tendermint.v1.ClientState provider_client_state = 5;
   // ProviderConsensusState filled in on new chain, nil on restart.
   ibc.lightclients.tendermint.v1.ConsensusState provider_consensus_state = 6;
-  // MaturingPackets nil on new chain, filled on restart.
+  // MaturingPackets nil on new chain, filled in on restart.
   repeated MaturingVSCPacket maturing_packets = 7
       [ (gogoproto.nullable) = false ];
   // InitialValset filled in on new chain and on restart.
   repeated .tendermint.abci.ValidatorUpdate initial_val_set = 8
       [ (gogoproto.nullable) = false ];
-  // HeightToValsetUpdateId nil on new chain, filled on restart.
+  // HeightToValsetUpdateId nil on new chain, filled in on restart.
   repeated HeightToValsetUpdateID height_to_valset_update_id = 9
       [ (gogoproto.nullable) = false ];
-  // OutstandingDowntimes nil on new chain, filled on restart.
+  // OutstandingDowntimes nil on new chain, filled  in on restart.
   repeated OutstandingDowntime outstanding_downtime_slashing = 10
       [ (gogoproto.nullable) = false ];
-  // PendingSlashRequests filled in on new chain, nil on restart.
+  // PendingSlashRequests nil on new chain, filled in on restart.
  interchain_security.ccv.consumer.v1.SlashRequests pending_slash_requests = 11
  [ (gogoproto.nullable) = false ];
+  // LastTransmissionBlockHeight nil on new chain, filled in on restart.
```

Comment:
Is this being used for a zero height restart or a restart containing the current block height? Or both?

Reply:
If this genesis restore was being used on a zero height restart, then that might result in it taking a really long time to send out a distribution... not sure

Reply:
You're right. A zero height restart could take a problematic long time due to this line:
https://github.com/cosmos/interchain-security/blob/b10e13207a56dafcbfe81ccfb273211e738de334/x/ccv/consumer/keeper/distribution.go#L58
Actually `LastTransmissionBlockHeight` should never be greater than the current block height.